### DISCUSSION TITLE
IFrame Widget

### DISCUSSION THREAD

**Discussion by SebastianAtWork** [bluecaret/carettab#350] [Ideas and Feature Requests]
Hi, how possible would it be to add an iframe widget? I just bought a subscription because i thought Shapes would have that functionality (My bad for not reading, no problem). Everthing else is already really cool but without an iframe, its not really what I am looking for. I want to see my google calendar in a mobile view (doable with iframes) and my todoist todos.

**Comment by bluecaret** [comment]
Off the top of my head, I'm not sure if that will be allowed by the browser for extensions or not. But if it is allowed I will add that in, it is a great idea!

**Comment by bluecaret** [comment]
An IFrame widget will be available in an upcoming update, 4.2.0, that should be available in a few days. Please note that due to security settings that some sites add, not every site will work and this is unfortunately nothing I can do about it. Some examples I tested were "Google.com" and "Youtube.com" and unfortunately Google Calendar all didn't work when I tested them. However plenty of other websites should work.
